int64 arm_pos
int64 lin_act
bool drum_cont
bool drum_turn
bool reed_swt